config/300-googlecloudstoragesource.yaml

```diff
@@ -24,7 +24,10 @@ metadata:
   annotations:
     registry.knative.dev/eventTypes: |
       [
-        { "type": "com.google.cloud.storage.notification" }
```

Comment:
what happens if we have a generic event ? you don't want that part of the annotation ?

Reply:
This source does not have generic events, however, I brought it back for test scenarios or potential unhandled situations as a fallback option. Thanks!